### PR TITLE
[All] Cleaning empty json elements using REGEX like a pro

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Until now (https://github.com/KratosMultiphysics/Kratos/pull/10562) when generating results in JSON of empty values were added filling of garbage the json result files. This clean up that. I was cleaning by hand until I remembered that REGEX exists...

**🆕 Changelog**

- [Cleaning (I)](https://github.com/KratosMultiphysics/Kratos/commit/4523b69e1a277e2e6bacc422d458b4f1bd0261cf)
- [Using REGEX like a pro](https://github.com/KratosMultiphysics/Kratos/commit/91c5ba94bdb7db06a20d6f6fd6f64135182deae0)
- [Remove also empty nodes](https://github.com/KratosMultiphysics/Kratos/pull/10567/commits/65ec22091d5da0f0535b4668405c3c494fba702d)

### CODE REVIEW

File: applications/IgaApplication/tests/coupling_condition_tests/two_patch_refined_test/two_patch_refined_test_lagrange_membrane_result.json [hunk 1]
```diff
@@ -125,29 +125,29 @@
...
+    
+    
+    
+    
+    
+    
+    
+    
+    
     "ELEMENT_25": {}
```
Comment (matekelemen) [inline applications/IgaApplication/tests/coupling_condition_tests/two_patch_refined_test/two_patch_refined_test_lagrange_membrane_result.json:152]: line breaks are still there, and the last entries seem to be ignored

**Review by matekelemen — COMMENTED** [review]
https://stackoverflow.com/a/1732454/12350793 -- same for JSON; it's probably safer to just parse the JSONs, get rid of the junk, and write them anew.

Also, I doubt anyone will honestly review changes to ~200 single-line JSON files.

File: applications/IgaApplication/tests/coupling_condition_tests/two_patch_refined_test/two_patch_refined_test_lagrange_membrane_result.json (hunk 1 above)
Comment (loumalouomega) [reply applications/IgaApplication/tests/coupling_condition_tests/two_patch_refined_test/two_patch_refined_test_lagrange_membrane_result.json:152]: I cleaned that too

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]
To be discussed in @KratosMultiphysics/technical-committee

**Comment by RiccardoRossi** [comment]
@KratosMultiphysics/technical-committee is not positive about this large automatic refactorings unless there is an important reason. Our point of view is that the risk is not worth the advantage the change may bring onto the table as they may easily introduce bugs.

**Pull request closed without merging**